spec/pagseguro/items_spec.rb

```diff
@@ -36,16 +36,6 @@
     expect(items).to include(item)
   end
 
-  it "increases quantity when adding duplicated item" do
-    item = PagSeguro::Item.new(id: 1234)
-    items = PagSeguro::Items.new
-    items << item
-    items << item
```

Comment:
This is a poor test case, anybody see the api can try to do this:

```
item = PagSeguro::Item.new(id: 1234
store = PagSeguro::Items.new
store << item
store << { id: 1234 }
```

but the first item returns a wrong quantity.

:bomb::bomb::bomb: